LICENSE.md

```diff
@@ -0,0 +1,201 @@
+Content under `/apps` is provided under MIT License as follows:
+
+**MIT License**
+
+Copyright (c) 2023 Aggregate Intellect Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+=======
+
+Content under `/doc` is provided under Creative Common License as follows:
+
+**Creative Commons Legal Code**
+
+# Creative Commons Attribution-ShareAlike 4.0 International
```

Comment:
I think you can delete lines 31-41 inclusive. As I understand it this is explanatory preamble from Creative Commons, not the actual text of the license. 

You could also add a reference to the source e.g. "For more information about this license see https://creativecommons.org/licenses/by-sa/4.0/legalcode".